src/standalone/Soil/energy_hydrology.jl

```diff
@@ -745,9 +762,7 @@ Returns the surface albedo field of the
 `EnergyHydrology` soil model.
 """
 function ClimaLand.surface_albedo(model::EnergyHydrology{FT}, Y, p) where {FT}
-    return @. FT(
-        0.5 * model.parameters.PAR_albedo + 0.5 * model.parameters.NIR_albedo,
-    )
+    return @. FT(0.5 * p.soil.PAR_albedo + 0.5 * p.soil.NIR_albedo)
```

Comment:
```suggestion
    return @.  (p.soil.PAR_albedo + p.soil.NIR_albedo) / 2
```
so that you don't have to worry about the FT